Bug?

Was attempting to do some tweaks to my github actions and ran across this error during build...

```
Running /CustoPiZer/start_chroot_script in chroot...
Building on non-ARM device a armv7l system, using qemu-arm-static
+ set -e
+ export LC_ALL=C
+ LC_ALL=C
+ source /common.sh
+ install_cleanup_trap
+ set -e
+ trap cleanup SIGINT SIGTERM
+ '[' raspbian == ubuntu ']'
+ '[' -n '' ']'
+ '[' -n '' ']'
+ apt-get update
Get:1 http://archive.raspberrypi.org/debian buster InRelease [32.6 kB]
Get:2 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
Get:3 http://archive.raspberrypi.org/debian buster/main armhf Packages [378 kB]
Reading package lists...
E: Repository 'http://raspbian.raspberrypi.org/raspbian buster InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
Error: Process completed with exit code 100.
```

I'm assuming this is related to debian release change upstream from raspbian based on this article.

https://www.raspberrypi.org/forums/viewtopic.php?p=1905664

--- a/src/start_chroot_script
+++ b/src/start_chroot_script
@@ -59,7 +59,7 @@ then
   cat /home/"${EDITBASE_USER}"/.pydistutils.cfg
 fi
 
-apt-get update
+apt-get update --allow-releaseinfo-change
 if [ $( is_in_apt policykit-1 ) -eq 1 ]; then
   sudo apt-get -y install policykit-1
 fi

